packages/connect-popup/e2e/tests/permissions.test.ts

```diff
@@ -18,23 +19,49 @@ const fixtures = [
     {
         description: `iframe and host same origins but with trust-issues=true -> show permissions`,
         queryString: '?trust-issues=true',
+        setTrustedHost: false,
         expect: () => popup.waitForSelector("[data-test='@permissions/confirm-button']"),
     },
     {
         description: `iframe and host same origins but with trust-issues=false -> no permissions shown`,
         queryString: '?trust-issues=false',
+        setTrustedHost: false,
         expect: () =>
             popup.waitForSelector('//p[contains(., "Follow instructions on your device")]', {
                 state: 'visible',
                 strict: false,
             }),
     },
+    {
+        description:
+            'iframe and host same origins but with settings to trustedHost -> no permissions shown',
+        queryString: '',
+        setTrustedHost: true,
+        trustedHost: true,
```

Comment:
there is one extra line? `setTrustedHost` and `trustedHost`, probably some leftover

Reply:
yes, not needed 4b6ffdeaf003b86db13a6fd0c5a85607be70e6f9